pkg/kubectl/history.go

```diff
@@ -67,22 +67,22 @@ func (h *DeploymentHistoryViewer) History(namespace, name string) (HistoryInfo,
 	if err != nil {
 		return historyInfo, fmt.Errorf("failed to retrieve deployment %s: %v", name, err)
 	}
-	_, allOldRCs, err := deploymentutil.GetOldReplicaSets(*deployment, h.c)
+	_, allOldRSs, err := deploymentutil.GetOldReplicaSets(*deployment, h.c)
```

Comment:
This was an artifact of the weird commits ordering, which itself was due to an incorrect rebase. Fixed the rebase and squashed the commits. This has disappeared now.